lib/src/player_with_controls.dart

```diff
@@ -36,7 +36,7 @@ class PlayerWithControls extends StatelessWidget {
       return Stack(
         children: <Widget>[
           if (chewieController.placeholder != null)
-            chewieController.placeholder!,
+            _buildPlaceholder(chewieController, context) ?? Container(),
```

Comment:
Remove the placeholder `null` check since it's redundant:

```dart
children: <Widget>[
_buildPlaceholder(chewieController, context) ?? Container(),
...
]
```